src/components/RegistrationForm/RegistrationForm.jsx

```diff
@@ -24,39 +24,112 @@ import walletSVG from '../../assets/icons/wallet.svg';
 import emailSVG from '../../assets/icons/email.svg';
 import passwordSVG from '../../assets/icons/lock.svg';
 import nameSVG from '../../assets/icons/profile-name.svg';
-
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
 import { Link } from 'react-router-dom';
+import { passwordStrength } from 'check-password-strength';
+import PasswordStrength from '../PasswordStrength/PasswordStrength';
+
+const SignupSchema = Yup.object().shape({
+  email: Yup.string().email('Invalid email').required(''),
```

Comment:
Sprawdź czy można jakoś to zindywidualizować bo w tej chwili akceptuje adres e-mail bez końcówki z domeną np:
nie wyświetla błędu przy:
Adam@gmail